config/db.flanksource.com_elasticsearchdbs.yaml

```diff
@@ -0,0 +1,76 @@
+
+---
```

Comment:
This file looks like it duplicated ?

Reply:
controller gen generated two copies.  Wasn't sure if the second is necessary